src/routes/PageLayout.tsx

```diff
@@ -118,18 +119,28 @@ export const PageLayout: FC = () => {
           height={requiredSnapshot ? 'calc(100vh - 3rem)' : '100vh'}
         />
         <Box w="100%">
-          <ScaleFade in={true} key={location.key}>
-            <Flex
+          <ScaleFade
+            in={true}
+            key={location.key}
+            style={{
+              display: 'flex',
+              width: '100%',
+              height: '100%',
+              justifyContent: 'center',
+              padding: '2.5rem 2rem 0',
+              paddingBottom:
+                location.pathname === ROUTES.ACCOUNT ? '0' : '2.5rem',
```

Comment:
These route-specific params don't feel great, but I can't think of a clean way to do this that fits with how the PageLayout component is used. IMO, PageLayout is probably doing too much.